common/lib/xmodule/xmodule/course_module.py

```diff
@@ -303,6 +303,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
```

Comment:
To be clear, we keep it FB-specific throughout.  What I'm pointing out, is that the API envelopes the FB-specific URL within a social structure.